getting-started.html

```diff
@@ -69,13 +69,33 @@
         <div class="col-sm-8 wrapper">
           <h1 class="page-header">Getting Started</h1>
 
-          <h2>My First Steps with Interledger</h2>
+          <h2>Sending Your First Interledger Payment</h2>
 
-          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+          <p class="intro">Sign up for the demo wallets and start sending Interledger payments between them in under 3 minutes.</p>
 
-          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+          <p>First, create an account on the <a target="_blank"href="https://red.ilpdemo.org">Red Demo Wallet</a> and the <a target="_blank"href="https://blue.ilpdemo.org">Blue Demo Wallet</a> (one for sending and the other for receiving).</p>
 
-          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+          <div>
+          <img src="./assets/demo_register.png" />
```

Comment:
@emschwartz I believe we can reference assets with just ` <img src="assets/demo_register.png" />` Does that work?